Ghost_in_the_Shell/client_folder_react_app/ethisim/src/post.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import { baseURL } from '../../Constants/Config'
+// Universal fetch request using axios
+export default function universalFetch(setResponse, endpoint, onError, onSuccess) {
+    console.log('Fetch started');
```

Comment:
We need extra parameter of "requestBody" for POST request.

Reply:
done